packages/api-explorer/src/components/DocPseudo/DocPseudo.tsx

```diff
@@ -39,10 +41,12 @@ export const DocPseudo: FC<PseudoProps> = ({ method }) => {
 
   return (
     <Flex alignItems="flex-start">
-      {`${method.operationId}(`}
-      <DocParams parameters={args} />
-      {'): '}
-      <DocPrimaryResponse response={resType} />
+      <Code fontSize="small" fontWeight="normal">
```

Comment:
`ApixCode` does not fit here. We need to rethink whether we need that to be a global override. cc @miketorosian @jkaster

Reply:
I think the "pseudocode" render may have specific styling rules that don't apply elsewhere due to the tooltips, wrapping of parameters, and relatively compact render.